### DISCUSSION TITLE
Build stucked on queued

### DISCUSSION THREAD

**Discussion by daviddomkar** [codemagic-ci-cd/codemagic-docs#1752] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Run a build from Codemagic UI or through a webhook using codemagic.yaml configuration.

### Expected results

I expect the build to start executing.

### Actual results

The build is stucked on queued. I waited for 20 minutes, canceled the build, tried again, still stucked on queued. I tried reverting codemagic yaml to the state when it worked. Still stucked on queued.

### Build id (optional)

6484e0b34b3f7553efd727da

**Comment by diegoalex** [comment]
Same issue here, queued for over 1 hour already.
I've tried to cancel and start again but no success.

Build ID: 6484ebda4da83b1e596cff09

**Reply by diegoalex** [reply]
My first build has just started. 🙌🏼

**Reply by diegoalex** [reply]
Started but got stuck in preparing .. I've canceled the builds.
will try tomorrow.

**Reply by diegoalex** [reply]
The issue seems to be resolved. Builds are running as usual today.

**Comment by simonkoeck** [comment]
Same here, my build is also stuck on queued :/

**Comment by Clon1998** [comment]
Same for me.
Scheduled a build about 2hrs ago. Just noticed it was still queued. Canceled and rescheduled without luck.
Ids:
6484f2664da83b1e596cfff2
6484d55ea2b3f973630470ee

**Comment by NinoBass** [comment]
Also having this issue guys,
All my builds status are defaulting to queued, canceling and restarting doesn't fix this as well.
Please, someone should do well to notify me via this thread once this issue is resolved

**Comment by iamngoni** [comment]
Experiencing the same issue. What's up 😅

**Comment by Sa3dy** [comment]
I have the same issue over 2 hours ago

**Comment by jschaeken** [comment]
Same issue here, thought my account was the issue but tried a second one, still all queued. Flutter App via workflow editor.

**Comment by ted-nz** [comment]
The first build in the queue "started" but seem to be immediately stuck with the "preparing" status, been this way for over an hour.

**Comment by shuto-mito** [comment]
This issue appears to have been resolved.

**Comment by adnanjpg** [comment]
I left my app in a `queued` status and went to sleep, woke up to see it got stuck there for 4 hours. I'm on free tier, will that be deducted from my monthly, if so then that's going to be half of my monthly build time

**Reply by icarusdust** [reply]
Hi @adnanjpg, no, it won't consume your build minutes.

**Reply by jingp-ofload** [reply]
@icarusdust is it possible for your guys to publish the service status somewhere?
